components/search/list-renderer/analysis-set.js

```diff
@@ -20,7 +21,14 @@ export default function AnalysisSet({ item: analysisSet }) {
   const summary = analysisSet.summary;
   const inputFileSetsAccessions =
     analysisSet.input_file_sets?.length > 0
-      ? analysisSet.input_file_sets.map((fileSet) => fileSet.accession)
+      ? analysisSet.input_file_sets
+          .map((fileSet, index) => [
+            <Link key={fileSet["@id"]} href={fileSet["@id"]}>
+              {fileSet.accession}
+            </Link>,
+            index < analysisSet.input_file_sets.length - 1 && ", ",
+          ])
+          .flat()
```

Comment:
You can do this in an easier and more standardized way using the [SeparatedList component](https://github.com/IGVF-DACC/igvf-ui/blob/dev/components/separated-list.js). We have examples all over the code base. You can just collect up all the input file sets and flatten them into one array, then put a map loop within `<SeparatedList>`. Probably easier to do the render part inline with `<SearchListItemSupplementContent>`.

Reply:
Replaced.